### PR TITLE
Fixed CI and added indicator.

### PR DESCRIPTION
**Pull request by blackgnezdo**
Sorted test names to make rebuilding meaning.

### CODE REVIEW

**Comment by gunan** [comment]
Jenkins, test this please.

**Comment by gunan** [comment]
Jenkins, test this please.

**Comment by gunan** [comment]
Jenkins, test this please

**Comment by gunan** [comment]
Jenkins, test this please

**Comment by gunan** [comment]
Jenkins, test this please

**Comment by blackgnezdo** [comment]
Jenkins, test this please

File: README.md [hunk 1]
```diff
@@ -1,3 +1,5 @@
+[![Build Status](https://ci.tensorflow.org/buildStatus/icon?job=tensorflow-haskell-master)](https://ci.tensorflow.org/job/tensorflow-master-cpu)
```
Comment (fkm3) [inline README.md]: Should this be http://ci.tensorflow.org/job/tensorflow-haskell-master?

Comment (blackgnezdo) [inline README.md]: Done.

**Comment by blackgnezdo** [comment]
#12 should be fully addressed.

@fkm3 you get the honor to merge it once everything is clear!

**Pull request merged**